terraform/workspace_variables/production.tfvars.json

```diff
@@ -0,0 +1,26 @@
+{
+  "environment_name": "production",
+  "key_vault_name": "s165p01-kv",
```

Comment:
see here https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/45/files#r881427525